indexer/utils.go

```diff
@@ -0,0 +1,26 @@
+package indexer
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow/protobuf/go/flow/access"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+func GetChain(ctx context.Context, accessURL string) (flow.Chain, error) {
```

Comment:
Not sure what the usage of this will be, but if you plan to use this to return the chain ID needed in the Ethereum RPC API, then know it's not the correct chain ID. The EVM uses two chain IDs 666 for testnet and 777 for mainnet, which you should configure once you start the service as a configuration option and then read it from that not from the network. You could however have some additional logic to detect the Flow Chain ID and compare that to EVM chain ID and for cases like mainnet you can make sure the correct EVM chain ID was set but that's way ahead.

Reply:
This is taken over from https://github.com/peterargue/execdata-client/blob/main/client/util.go#L13-L26, as I was not able to directly add it as a dependency, due to some conflicts with `go-ethereum` and `flow-go` versions. This is merely used to select on which network the indexer will connect to, `testnet` or `mainnet`.